docker/pytorch/Dockerfile

```diff
@@ -102,10 +84,22 @@ RUN pip install --no-cache-dir --find-links https://download.pytorch.org/whl/tor
 #########
 RUN rm -rf /tmp/*
 
+
+################################################
+# Create a virtualenv and activate it by default
+################################################
+RUN python${PYTHON_VERSION} -m venv --system-site-packages ~/mosaicml-venv && \
+    python${PYTHON_VERSION} -m venv --upgrade ~/mosaicml-venv && \
+    echo "source ~/mosaicml-venv/bin/activate" >> ~/.bashrc
```

Comment:
Will this trigger if you are not running a command under `/bin/bash`? I've usually seen people just do modify `PATH` to include `<path-to-venv>/bin`

Reply:
Ah kk, how would I do that in a shell-independent manner?

Reply:
Switched to this technique. But I am modifying the /etc/bashrc file as that is sourced by default(adding it to /etc/profile unfortunately doesn't seem to work)